src/components/Layout/Layout.js

```diff
@@ -16,15 +18,64 @@ import Footer from '../Footer';
 injectTapEventPlugin();
 
 class Layout extends React.Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+    isLoggedIn: PropTypes.bool.isRequired,
+  };
+
+  state = {
+    drawerOpen: true,
+  };
+
+  handleDrawerToggle = () => {
+    this.setState({ drawerOpen: !this.state.drawerOpen });
+  };
 
   render() {
+    if (!this.props.isLoggedIn) {
+      return (
+        <MuiThemeProvider muiTheme={this.context.muiTheme}>
+        <div>
+          <Header
+          />
+          {this.props.children}
+          {// <Feedback />
+          }
+          <Footer
+          isLoggedIn={this.props.isLoggedIn}
+          />
+        </div>
+        </MuiThemeProvider>
+      );
+    }
+    const contentStyle = {  transition: 'margin-left 450ms cubic-bezier(0.23, 1, 0.32, 1)' };
+
+    if (this.state.drawerOpen) {
+      contentStyle.marginLeft = 256;
+    }
+
     return (
       <MuiThemeProvider muiTheme={this.context.muiTheme}>
-        <div>
-          <Header />
+      <div>
+        <div style={contentStyle}>
+          <Navigation 
```

Comment:
Can you add Navigation and DrawerItems to Header? This way, all parts of the website get the Drawer and you can reduce the number of lines of code in Layout.js drastically. (Conditional rendering won't be required)